### ISSUE TITLE
Ignore bad lines instead of crashing

### ISSUE THREAD

**Issue by tobiasmuehl** [BurntSushi/xsv#304]
When a CSV file is formatted incorrectly, xsv will crash. It would be very handy if there is a parameter that excludes those lines from processing, but continues processing the rest of the file

**Comment by BurntSushi** [comment]
Am I just supposed to take your word for it? :-) Could you please show an example?

`xsv` should never crash on any input.

It might return errors in some cases. And what to do in those cases depends on what you're trying to do. But you didn't give any examples nor did you describe what you're trying to do nor the output of `xsv` when it crashed.

(If it were possible, I would love to have a `--please-no-bugs` that one could pass to any program to prevent it from crashing. :P)

**Comment by tobiasmuehl** [comment]
My apologies, I picked the wrong word. `xsv` indeed doesn't crash - it exits with an error, which is correct, as my input CSV file is malformed (too many columns). In looking into this problem I stumbled upon [csvclean from csvkit](https://csvkit.readthedocs.io/en/1.0.2/scripts/csvclean.html) which can be used to fix/remove bad lines from my CSV file. Given that `xsv` already has the fixlengths command, perhaps adding a flag to skip lines with errors wouldn't be too much trouble?
